machine_management/adding-rhel-compute.adoc

```diff
@@ -0,0 +1,39 @@
+[id="adding-rhel-compute"]
+= Adding RHEL compute machines to an {product-title} cluster
+include::modules/common-attributes.adoc[]
+:context: adding-rhel-compute
+
+toc::[]
+
+In {product-title}, you can add Red Hat Enterprise Linux (RHEL) compute, or worker, machines to a user-provisioned infrastructure cluster. You can use RHEL as the operating system on only compute machines.
```

Comment:
> ... to a user-provisioned infrastructure cluster
Looks like this is for UPI only, but per BZ 1872580, it should be for UPI and IPI, right?